.travis.yml

```diff
@@ -1,39 +1,26 @@
 language: cpp
-compiler: clang
 
-before_install:
-#  - if [ "$TRAVIS_MCSERVER_BUILD_TYPE" == "COVERAGE" ]; then sudo pip install cpp_coveralls; fi
+compiler: 
+- clang
+- gcc
 
-  # g++4.8
+before_install:
   - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
   - sudo apt-get update -qq
 
 install:
-  # g++4.8 and clang
   - sudo apt-get install -qq g++-4.8
+  - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.8 90
```

Comment:
You need to update gcc as well, or else we could get weird errors from using a different gcc/g++ version.